### PR TITLE
Add JDBC Project artifacts

### PR DESCRIPTION
**Pull request by aahmed-se**

### CODE REVIEW

**Comment by danosipov** [comment]
👍

File: jdbc/build.gradle [hunk 1]
```diff
@@ -0,0 +1,4 @@
+dependencies {
+    compile 'com.zaxxer:HikariCP:2.7.2'
```
Comment (Antwnis) [inline jdbc/build.gradle:2]: Never used it before, but  HikariCP looks nice indeed

File: build.gradle [hunk 2]
```diff
@@ -124,3 +124,10 @@ project(':kafka') {
 		compile project(':core')
 	}
 }
+
+//JDBC
+project(':jdbc') {
```
Comment (Antwnis) [inline build.gradle:129]: Is this a JDBC **Source** or a **Sink** or a **CDC** - Change Data Capture? or is it a **commons** for all JDBC related stuff?

Comment (aahmed-se) [reply build.gradle:129]: well initially just a simple source and sink

**Pull request merged**